packages/menu/src/MenuGroup.ts

```diff
@@ -41,13 +46,24 @@ export class MenuGroup extends SpectrumElement {
         MenuGroup.instances += 1;
     }
 
+    private get menuRole() {
```

Comment:
We'll need to confirm that this actually associates the menu with its items through the slot... `sp-action-group` has a slightly different relationship between the `group` and `item` elements.